### DISCUSSION TITLE
获取component组件路径

### DISCUSSION THREAD

**Discussion by WT-Road** [umijs/umi#10597] [Q&A] [answered]
想根据路由path获取对应的component组件路径
找了一圈没有对应的APi
请问有办法能获取到吗
![image](https://user-images.githubusercontent.com/14371925/221101931-18da1622-f42c-4802-a3dd-a39a1995843f.png)

**Comment by fz6m** [comment, marked as answer]
这是内部数据，尝试在 [`useAppData`](https://umijs.org/docs/api/api#useappdata) 、`import { getRoutes } from '@@/core/route'` 里寻找答案
